docs/MD.md

```diff
@@ -1,29 +1,129 @@
+  - Example phrases are followed by their translations. If the example ends in punctuation, the translation simply follows the example, after a space. If the example does not end in punctuation, the example and its translation are separated by a hyphen surrounded by spaces.
+  - **ex:** "You are cold. Usually used in a question. E.g. kikawacin ci? Are you cold?"
+  - **ex:** "Ahead of time. Now, instead of later. E.g. kisac tota - Do it now, instead of later. Also, at once."
+* Some of the definitions have Cree words in them, and we'd like these converted to cross-references, and their spellings standardized to SRO. This will probably require manual editing.
+  - **ex:** https://itwewina.altlab.app/search?q=ahci: supposed to be _âhc âna_ (from _âhci ana_)
 * The definitions sometimes contain encyclopedic / usage notes. These are not demarcated in any special way, except perhaps as distinct sentences.
-* The definitions include sentence punctuation, which should not be sent to the FST. They do not include abbreviations such as `s.o.`.
+* The definitions include sentence punctuation, which should not be sent to the FST. The definitions do not include abbreviations such as `s.o.`.
 * Parentheticals = sentences that start with:
   - Also
   - And
   - Or
-* Grammatical information: `Animate.` | `Inanimate.`
 * Senses / definitions are occasionally separated by sense numbers.
-* Part of speech: `Noun.` | `noun.` | `verb.`
-* Cross-references: Any Cree words in the definition need to be flagged as cross-references.
+* Grammatical information: `Animate.` | `Inanimate.` (sometimes lowercase)
+* Part of speech: `noun.` | `verb.` (sometimes with initial capital letter)
 * We need to store 3 versions of the definition:
   - _original_: the original MD definition, verbatim
     - Store this in the alternative analysis entry.
   - _FST_: the cleaned and edited version sent to the FST
     - Store this in the main entry, on the sense from MD.
   - _user_: original, except with Cree words corrected
     - Store this in the main entry.
+
+## RapidWordsClasses
+
+A semicolon-delimited list of rapid word classes that apply to this entry. Each class corresponds to a index in the [RapidWordsIndices](#RapidWordsIndices) field.
+
+## RapidWordsIndices
+
+A semicolon-delimited list of indices for the rapid word classes that apply to this entry. Each index corresponds to a rapid words class in the [RapidWordsClasses](#RapidWordsClasses) field.
+
+## English_POS
+
+The English definition, with each word tagged for part of speech.
+
+* The tag is appended to the end of each word in the format `_{TAG}`, **ex:** `The_DET`, indicating that the word _The_ is a determiner.
+* Items with no tag have the appended underscore but no tag.
+* Punctuation is also tagged with just the underscore, and no tag.
+
+## English_Search
+
+Contains a simplified version of the definition. The use of this field and how it was generated is currently unclear.
```

Comment:
For verbs, this means providing an English-like entry starting with the infinitive form of the verb (using the lemma, and excluding the initial subject pronoun).